backend/doc/decisions/6-use-fastify-server.md

```diff
@@ -0,0 +1,3 @@
+Chosen [Fastify](https://www.fastify.io/) as a Node web server over [Express](https://expressjs.com/) because the latter
```

Comment:
Curious if you've used Fastify in production before. This is the first I've heard of it.

Reply:
Before, I didn't use Node.js nor PostgreSQL, neither in production nor at all :)

Reply:
I don't have particular reasons to use Fastify other than mentioned: support for async handlers out of the box. It's "speed" is certainly not an argument.

Also, Express project looks [mostly abandoned](https://github.com/expressjs/express/graphs/contributors).

Reply:
Copy that!